build-system/tasks/update-packages.js

```diff
@@ -79,6 +79,12 @@ function patchWebAnimations() {
   // See https://github.com/ampproject/amphtml/issues/18612 and
   // https://github.com/web-animations/web-animations-js/issues/46
   file = file.replace(/b.true=a/g, 'b?b.true=a:true');
+
+  // Fix web-animations-js code that attempts to write a read-only property.
+  // https://github.com/ampproject/amphtml/issues/19783 and
+  // https://github.com/web-animations/web-animations-js/issues/160
+  file = file.replace(/this\._isFinished\s*=\s*!0,/g, '');
```

Comment:
Doesn't `!` also need an escape?

Also, for safety, there's only one such instance. Should we remove `/g`?

Reply:
done, ptal.